File: Phasegate/DayOne/Level 1/python/TEST3/test_task_three.py
from unittest import TestCase
import task_three
class task_three_test(TestCase):
    def test_that_perfect_squares_are_returned(self):
        numbers  = [4, 7, 9, 10, 16, 18]
        actual   = task_three.perfect_squares(numbers)
        expected = [4, 9, 16]
        self.assertEqual(actual, expected)

File: Phasegate/DayOne/Level 1/java/TEST3/TestTaskThree.java
import org.junit.jupiter.api.Test;
import static org.junit.jupiter.api.Assertions.*;
public class TestTaskThree{
    @Test
    public void testPerfectSquares(){
        int[] number   = {4, 7, 9, 10, 16, 18};
        int[] expected = {4, 9, 16};
        int[] actual   = TaskThree.perfectSquares(number);
        assertArrayEquals(expected, actual);
    }
}
